packages/protocol/script/layer1/deploy_protocol_on_l1.sh

```diff
@@ -3,44 +3,26 @@
 # This script is only used by `pnpm test:deploy:l1`.
 set -e
 
-export PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+export PRIVATE_KEY=0xbcdf20249abf0ed6d944c0288fad489e33f66b3960d9e6229c1cd214ed3bbe31
```

Comment:
what is this private key, and why it makes sense to have it in plain text here. I see taiko had one as well.

Reply:
This is a test private key on kurtosis devnet. This script is only going to be used in test deployment within Kurtosis package.